[
    {
        "user": "U3A8Z1QCC",
        "text": "<@U3A8Z1QCC|mylara> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1480753580.005951"
    },
    {
        "type": "message",
        "user": "U2R9D1FGD",
        "text": "When i deploy chaincode , A problem occured ,  \"11:32:56.835 [chaincode] Launch -&gt; ERRO 24d sending init failed(handler not found for chaincode jdoe)\n\" .How can I solve it ? Thank you",
        "ts": "1480765065.005953"
    },
    {
        "type": "message",
        "user": "U2R9D1FGD",
        "text": "There is no \"peer network login &lt;username&gt;\"",
        "ts": "1480765104.005954"
    },
    {
        "type": "message",
        "user": "U317M1YJU",
        "text": "hello, I have a problem with running chaincode_example02. After successfully running docker-compose up and building chaincode_example02 I'm trying to run CORE_CHAINCODE_ID_NAME=mycc CORE_PEER_ADDRESS=0.0.0.0:7051 .\/chaincode_example02 and getting the error \"CORE_CHAINCODE_ID_NAME is not recognized as commandlet...\"",
        "ts": "1480782268.005956"
    },
    {
        "user": "U3AAY7PPX",
        "text": "<@U3AAY7PPX|lesterpig> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1480784791.005957"
    },
    {
        "user": "U3A72F9RR",
        "text": "<@U3A72F9RR|edrian> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1480794976.005959"
    },
    {
        "user": "U3A9YAPGA",
        "text": "<@U3A9YAPGA|richardo> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1480811513.000003"
    },
    {
        "user": "U3B0TNS5D",
        "text": "<@U3B0TNS5D|david.druker> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1480812909.000004"
    },
    {
        "user": "U3A83MMK5",
        "text": "<@U3A83MMK5|steven.landers> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1480814596.000005"
    },
    {
        "type": "message",
        "user": "U3A83MMK5",
        "text": "is there any special trick to getting certificate attributes when using dockerized chaincode (not --peer-chaincodedev)?   I see the attributes on the cert - membership services says ACA is enabled.",
        "ts": "1480814967.000006"
    },
    {
        "type": "message",
        "user": "U3A83MMK5",
        "text": "i added a stackoverflow question with some debugging info here: <http:\/\/stackoverflow.com\/questions\/40949391\/not-seeing-certificate-attributes-when-not-running-locally>",
        "attachments": [
            {
                "title": "Not seeing certificate attributes when not running locally",
                "title_link": "http:\/\/stackoverflow.com\/questions\/40949391\/not-seeing-certificate-attributes-when-not-running-locally",
                "text": "The ProblemWhen I deploy a 4 peer nodes with PBFT or NOOPS in the cloud, any user certificate attributes are not seen. The values are blank.Observations Everything works locally. This suggest...",
                "fallback": "Not seeing certificate attributes when not running locally",
                "thumb_url": "http:\/\/cdn.sstatic.net\/Sites\/stackoverflow\/img\/apple-touch-icon@2.png?v=73d79a89bded&a",
                "from_url": "http:\/\/stackoverflow.com\/questions\/40949391\/not-seeing-certificate-attributes-when-not-running-locally",
                "thumb_width": 316,
                "thumb_height": 316,
                "service_icon": "http:\/\/cdn.sstatic.net\/Sites\/stackoverflow\/img\/favicon.ico?v=4f32ecc8f43d",
                "service_name": "stackoverflow.com",
                "id": 1
            }
        ],
        "ts": "1480815032.000007"
    },
    {
        "type": "message",
        "user": "U3A83MMK5",
        "text": "<@U317M1YJU> i've had luck using the node sdk from within a docker container to deploy chaincode, if that helps",
        "ts": "1480815191.000009"
    },
    {
        "user": "U3B10RHTQ",
        "text": "<@U3B10RHTQ|camelsoft> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1480826242.000011"
    },
    {
        "user": "U3B17J9E2",
        "text": "<@U3B17J9E2|iyawat> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1480832749.000012"
    },
    {
        "type": "message",
        "user": "U3A83MMK5",
        "text": "(Got it figured out.  I had mismatched fabric versions on fabric-baseimage vs. peer vs. node sdk)",
        "ts": "1480834756.000013"
    },
    {
        "user": "U3AD3B19S",
        "text": "<@U3AD3B19S|jotege> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1480836004.000014"
    }
]